packages/docs/src/stories/templates/header.stories.ts

```diff
@@ -73,7 +73,7 @@ export const SampleA = {
       </div>
     </sd-header>
     <sd-drawer id="sample-a-drawer" placement="end">
-      <nav>
+      <nav class="-mx-4">
```

Comment:
Since this changes the styling of the drawer, I think it's best to sync with @MartaPintoTeixeira to double check that this how it really should look like. Same for the change in L85 for the footer and the other templates below.